packit/api.py

```diff
@@ -1078,6 +1092,72 @@ def get_default_commit_description(
             resolved_bugs=resolved_bugs_msg,
         )
 
+    def get_release_link(self, upstream_tag: str, version: str) -> str:
+        """
+        Get link to the release via API if the GitProject is initialised.
+        """
+        if not self.up.local_project.git_project:
+            return ""
+
+        release = self.up.local_project.git_project.get_release(
+            tag_name=upstream_tag,
+            name=version,
+        )
+        return release.url if release else ""
+
+    def get_pr_description(
+        self,
+        upstream_tag: str,
+        version: str,
+        release_monitoring_project_id: Optional[int] = None,
+        resolved_bugs: Optional[list[str]] = None,
+    ) -> str:
+        """
+        Get the description used in pull requests for syncing release.
+        """
+        resolved_bugzillas_info = ""
+        if resolved_bugs:
+            for bug in resolved_bugs:
+                bug_split = bug.split("#")
+                if len(bug_split) == 2:
+                    bug_id = bug_split[1]
```

Comment:
I wonder if regex wouldn't be better here (to verify also that the ID is numeric) in order not to generate invalid links.